From 6391b468b1cf1e95b1f8020fadc59733c510eab6 Mon Sep 17 00:00:00 2001
From: Jan Petrous <jan.petrous@nxp.com>
Date: Mon, 12 Sep 2022 10:49:46 +0200
Subject: [PATCH 102/102] s32g: hwconfig: Unify 'xpcs_mode' for S32G

All devboards has PFE_EMAC_0 mapped to SGMII,
on EVB/EVB3 connected to 2.5G PHY AQR, on RDB2/RDB3
connected to SJA1110 automotive switch.

Use 2.5G speed on PFE_EMAC_0 by default for all boards.

Issue: ALB-9290
Upstream-Status: Pending 

Signed-off-by: Jan Petrous <jan.petrous@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 board/nxp/s32-cc/s32g/Kconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/board/nxp/s32-cc/s32g/Kconfig b/board/nxp/s32-cc/s32g/Kconfig
index 2c17c3c47a..7cc79d5d88 100644
--- a/board/nxp/s32-cc/s32g/Kconfig
+++ b/board/nxp/s32-cc/s32g/Kconfig
@@ -42,5 +42,4 @@ config NXP_S32GEVB_BOARD
 	select SPI_FLASH_MACRONIX
 
 config S32GEN1_HWCONFIG
-	default "pcie0:mode=rc,clock=ext;pcie1:mode=sgmii,clock=ext,fmhz=125,xpcs_mode=2G5" if (PCIE_S32GEN1 && FSL_PFENG) && (NXP_S32GRDB_BOARD || TARGET_S32G3XXAEVB || TARGET_S32G3XXAEVB3)
-	default "pcie0:mode=rc,clock=ext;pcie1:mode=sgmii,clock=ext,fmhz=125,xpcs_mode=0" if (PCIE_S32GEN1 && FSL_PFENG) && TARGET_S32G2XXAEVB
+	default "pcie0:mode=rc,clock=ext;pcie1:mode=sgmii,clock=ext,fmhz=125,xpcs_mode=2G5"
-- 
2.17.1

